templates/bundle_version.sh

```diff
@@ -0,0 +1,16 @@
+GIT=$(sh /etc/profile; which git)
+GIT_RELEASE_VERSION=$(${GIT} describe --tags --always --abbrev=0)
+COMMITS=$(${GIT} rev-list HEAD | wc -l)
+COMMITS=$(($COMMITS))
+TARGET_PLIST="${TARGET_BUILD_DIR}/${INFOPLIST_PATH}"
+DSYM_PLIST="${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Info.plist"
+
+for PLIST in $TARGET_PLIST $DSYM_PLIST
+do
+  if [ -f "$PLIST" ] ; then
+    /usr/libexec/PlistBuddy -c "Set :CFBundleVersion ${COMMITS}" "$PLIST"
+    /usr/libexec/PlistBuddy -c "Set :CFBundleShortVersionString ${GIT_RELEASE_VERSION#*v}" "$PLIST"
+  fi
+done
+
+echo "Updated ${TARGET_BUILD_DIR}/${INFOPLIST_PATH}"
```

Comment:
I'm actually ditching the echo, since I don't think it's going to be super useful.